app_grim_reefer/src/main.c

```diff
@@ -3,20 +3,40 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+#include "buzzer.h"
+#include "config.h"
+#include "data_storage.h"
 
+#include <launch_core/dev/dev_common.h>
+#include <math.h>
 #include <zephyr/device.h>
 #include <zephyr/devicetree.h>
+#include <zephyr/drivers/adc.h>
 #include <zephyr/drivers/gpio.h>
-// #include <zephyr/fs/fs.h>
-// #include <zephyr/fs/littlefs.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/drivers/uart.h>
+#include <zephyr/fs/fs.h>
 #include <zephyr/kernel.h>
 #include <zephyr/logging/log.h>
+#include <zephyr/shell/shell.h>
 #include <zephyr/storage/flash_map.h>
-LOG_MODULE_REGISTER(main, CONFIG_APP_GRIM_REEFER_LOG_LEVEL_DBG);
+#include <zephyr/sys/base64.h>
+// Nasty, put these in the right place when you have time
+#include "ina260.h"
+
+#include <zephyr/../../drivers/sensor/lsm6dsl/lsm6dsl.h>
```

Comment:
Gross....

Reply:
Hella. Despite searching for a while I could not get any other way to get that file into the includes. It doesn't want to be found under the actual zephyr include directory but if you need the small footprint internal data structure you need the header :/

Reply:
Might need to take a closer look at this and what you mean when I have the time